lib/analyzer/static-analyzer.ts

```diff
@@ -44,6 +51,10 @@ export async function analyze(
     getNodeBinariesFileContentAction,
   ];
 
+  if (options.distroless) {
```

Comment:
something of a hack, especially as it appears twice in this file.
then again, the whole way we determine the package manager and dependencies is a bit hackish so I'm not feeling too bad with it right now.